Solutions/TransmitSecurity/SolutionMetadata.json

```diff
@@ -0,0 +1,16 @@
+{
+	"publisherId": "transmitsecurity",
+	"offerId": "transmitsecurity",
```

Comment:
Offer id should contain keyword sentinel. 

Please go through this readme for guidelines 

![image](https://github.com/Azure/Azure-Sentinel/assets/117061676/2c8735b6-0483-4a9d-92a9-3aa6e4fcb7c5)


https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/README.md

Reply:
Resolved. Changed it to 
	"offerId": "microsoft-sentinel-solution-transmitsecurity",
Thank you